apps/ffdweb-site/src/app/_constants/structuredDataConstants.ts

```diff
@@ -0,0 +1,43 @@
+import type { Organization, WithContext } from 'schema-dts'
+
+import {
+  BASE_URL,
+  FFDW_URLS,
+  ORGANIZATION_NAME,
+  ORGANIZATION_NAME_SHORT,
+} from '@/constants/siteMetadata'
+
+const { social, email } = FFDW_URLS
+
+export const SCHEMA_CONTEXT_URL = 'https://schema.org'
+export const SCHEMA_EVENT_ATTENDANCE_MODE_ONLINE_URL =
+  'https://schema.org/OnlineEventAttendanceMode'
+export const SCHEMA_EVENT_ATTENDANCE_MODE_OFFLINE_URL =
+  'https://schema.org/OfflineEventAttendanceMode'
+
+const DESCRIPTION =
+  'FFDW is a nonprofit organization committed to preserving humanity’s most important information by funding the development of open-source tools.'
```

Comment:
I think we should import this from `apps/ffdweb-site/src/app/(homepage)/page.tsx`